### DISCUSSION TITLE
Dump Metal Storm rerelease

### DISCUSSION THREAD

**Discussion by bmsuseluda** [sanni/cartreader#290] [Need help]
Hello together,

currently i received my copy of the rerelease of Metal Storm from retro-bit.
I tried to dump it with the settings of the original release http://nes.dnsabr.com/entity/view?id=1431 but wasn't successful.

Then i stumbled upon https://nerdlypleasures.blogspot.com/2020/01/product-review-retro-bits-metal-storm.html and they say that i have to use the mapper 286. Is this possible with this card reader?

Thanks a lot
bmsuseluda

**Comment by sanni** [comment]
The current NES module is based on skaman's Famicom Dumper (2019-08-31) and supports the following mappers:      
0, 1, 2, 3, 4, 5, 7, 9, 10, 13, 16, 18, 19, 21, 22, 23, 24, 25, 26, 32, 33, 34, 37, 47, 48, 65, 66, 67, 68, 69, 70, 71, 72, 73, 75, 76, 77, 78, 80, 82, 85, 86, 87, 88, 89, 92, 93, 94, 95, 96, 97, 105, 118, 119, 140, 152, 153, 154, 155, 159, 180, 184, 185, 206, 207, 210      
For any additional mapper like 286 you would need to add some lines to the current source code. 

So it all depends on your definition of possible. I myself have not looked at how NES mappers work yet. so I can't say if it's a lot of work or just a small modification.

**Reply by ApanApane** [reply]
Sorry to dig up an old thread. I was just wondering if there has been any progress on this issue in the last three years. Is the release dumpable now?

**Reply by PsychoFox11** [reply]
I'm guessing you mean mapper 268 rather than 286. The script for 268  isn't built-in as far as I know so far, though mappers over 255 are starting to have some limited support. It is possible with a script to dump 268. 2 of the 3 dumps (the 2 good ones, other is an overdump) of the Retro-Bit version on No-Intro list this as their dumper. And the valid header there specifies mapper 268.

**Reply by ApanApane** [reply]
That's interesting. I hadn't thought of checking No-Intro (rookie mistake, for sure) so thanks for pointing that out. Any guidance on how to dump it? I am not that tech savvy so I don't understand what "with a script" means in this context.

**Comment by nsx0r** [comment]
Metal Storm should be dumpable now, using mapper 268 and 512KiB PRG.

**Reply by ApanApane** [reply]
This might be a stupid question, but how do I select that mapper and PRG? I can't seem to select a mapper higher than 255. I am assuming that there is some software update I need to do that I am not aware of?

**Reply by nsx0r** [reply]
yeah I just pushed the update so it's only in the current code, not in the latest release package
